css/at-rules/media.json

```diff
@@ -1202,10 +1202,10 @@
                 "version_added": false
               },
               "opera": {
-                "version_added": false
+                "version_added": "61"
               },
               "opera_android": {
-                "version_added": false
+                "version_added": "54"
```

Comment:
#5281 _just_ added the data for version 54, so it's likely that your fork is slightly out of date. This should work, if you rebase (or merge) your branch with the latest `master`.